about/yocto-wpe.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: post
+title: "WPE Yocto images for Raspberry Pi"
+tags: [about]
+data: { dateless: "true" }
+permalink: /about/yocto-wpe.html
+---
+
+One way to easily test WPE is to use Yocto images for Raspberry Pi
+devices, write them in a micro-SD card and use them to boot the
+device.
+
+
+## Getting and flashing the image on a micro-SD card
+
+Use the following values:
+* **MACHINE**: one of **{raspberrypi3-mesa|raspberrypi4-64-mesa}**
+  * raspberrypi3-mesa is 32-bit, to be able to test WPE in a more resource-starved environment
+  * raspberrypi4-64-mesa is 64-bit
+* **SDIMG**: based on the machine, '`core-image-weston-browsers-${MACHINE}.wic`', e.g. '`core-image-weston-browsers-raspberrypi3-mesa.wic`'
```

Comment:
I think this should be written in a different way, maybe split in two different lines or structured differently, otherwise it's hard to read on the website:

![image](https://user-images.githubusercontent.com/2197259/211639787-5e5155f9-4d0a-44f5-9d7a-bff6bb0f8356.png)